projects/Draupnir/example.nix

```diff
@@ -38,12 +41,15 @@
   services.draupnir = {
     enable = true;
     settings = {
-      homeserverUrl = "http://localhost:8008";
+      homeserverUrl = "http://localhost:${builtins.toString port}";
```

Comment:
Quick tip: you can use some builtin functions without the `builtins` keyword and `toString` is one of them.